compiler/daml-extension/ci-tests.sh

```diff
@@ -2,13 +2,11 @@
 # Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-set -eux
-DAML_EXTENSION_DIR=$1
-CHECK_JSON=$2
+set -euo pipefail
 
-cd $DAML_EXTENSION_DIR
+cd compiler/daml-extension
 
 : Checking JSON files for proper syntax
 for sf in *.json syntaxes/*.json; do
-  $CHECK_JSON $sf
+  cat $sf | jq '.'
```

Comment:
Looks like this is failing because `jq` is not available. You probably need to add it to `data` and then get it via `$(location …)`.

Reply:
But it worked on my machine... 🤔

Reply:
Interestingly it also worked on CI on MacOS. Maybe `jq` is in the default bazel path there for some reason?

Reply:
And it succeeded on macOS. Is the sandboxing mechanism different (i.e. presumably inexistent) on macOS?

Reply:
I think there is still some level of sandboxing but the mechanism is definitely different.